redaxo/src/core/lib/console/config/get.php

```diff
@@ -17,6 +17,7 @@ protected function configure()
         $this->setDescription('Get config variables')
             ->addArgument('config-key', InputOption::VALUE_REQUIRED, 'config path separated by periods, e.g. "setup" or "db.1.host"')
             ->addOption('type', 't', InputOption::VALUE_REQUIRED, 'php type of the returned value, e.g. "octal"', 'string')
+            ->addOption('package', 'p', InputOption::VALUE_OPTIONAL, 'package to inspect, defaults to redaxo-core', 'core')
```

Comment:
```suggestion
            ->addOption('package', 'p', InputOption::VALUE_REQUIRED, 'package to inspect, defaults to redaxo-core', 'core')
```

(Optionen sind immer optional, aber wenn diese Option benutzt wird, dann ist ein Wert erforderlich)